azext_iot/central/_help.py

```diff
@@ -130,6 +131,27 @@ def _load_central_devices_help():
     """
 
 
+def _load_central_compute_device_key():
+    helps[
+        "iot central device compute-device-key"
+    ] = """
+          type: group
+          short-summary:  Generate a derived device SAS key
+      """
+
+    helps[
+        "iot central device compute-device-key create"
+    ] = """
+        type: command
+        short-summary: Generate a derived device SAS key.
```

Comment:
I'm still of the opinion that we should advertise that this can work fully offline

Reply:
central command  may have an online component in the near future so would like to not add offline comment here.